tests/PHPStan/Analyser/nsrt/preg_match_shapes.php

```diff
@@ -116,13 +116,6 @@ function doOffsetCapture(string $s): void {
 	assertType('array{}|array{array{string, int<0, max>}, array{string, int<0, max>}, array{string, int<0, max>}, array{string, int<0, max>}}', $matches);
 }
 
-function doUnmatchedAsNull(string $s): void {
```

Comment:
this pre-existing test no longer has the same expectations across all php versions.
therefore I moved it into the bug-11311 tests and adjusted expectations accordingly